tests/email/email_parsing_test.php

```diff
@@ -67,13 +67,11 @@ protected function setUp(): void
 		$phpbb_container->set('ext.manager', $extension_manager);
 
 		$log = new \phpbb\log\dummy();
-		$iconify_bundler = new \phpbb\assets\iconify_bundler($log, $phpbb_root_path);
-		$assets_bag = new \phpbb\template\assets_bag($iconify_bundler);
+		$assets_bag = new \phpbb\template\assets_bag();
 		$phpbb_container->set('assets.bag', $assets_bag);
```

Comment:
Probably unused

Reply:
Probably is unless I add back DI for this.

Reply:
Which I did now :)